modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+----
+
+. Create an `AgentServiceConfig` custom resource to install the Assisted Installer Service's controller
++
+[source,bash]
+----
+ cat <<EOF | oc create -f -
+ apiVersion: agent-install.openshift.io/v1beta1
+ kind: AgentServiceConfig
+ metadata:
+   name: agent
+   namespace: assisted-installer
+ spec:
+   databaseStorage:
+     volumeName: <db-pv-name> <1>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <db-storage-size> <2>
+   filesystemStorage:
+     volumeName: <fs-pv-name> <3>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <fs-storage-size> <4>
+   osImages:
+     - openshiftVersion: <ocp-version> <5>
+       rootfs: <rootfs-url> <6>
+       url: <iso-url> <7>
+ EOF
+----
+
+<1> PersistentVolume name to use for database storage.
+<2> Volume size of the `db-pv-name` PersistentVolume.
+<3> PersistentVolume name to use for file system storage.
+<4> Volume size of the `fs-pv-name` PersistentVolume.
+<5> OCP version to boot the SNO clusters with.
+<6> The rootfs ISO URL; part of step 4 of the prerequisites.
+<7> The image url; part of step 4 of the prerequisites.
+
+.Troubleshooting
+
+.Managed cluster fails to come online
+
+. Check the `AgentClusterInstall` install status:
+[source,terminal]
+----
+oc get agentclusterinstall -n <cluster-name>
+----
+
+If under the `INSTALLED` column is `false`, the installation was unsuccessful. Use `describe` to view error details.
+
+[source,terminal]
+----
+oc describe agentclusterinstall -n <cluster-name> <cluster-name>
+----
+
+. Amend the errors and do the following:
+
+Remove the cluster's namespace.
+[source,terminal]
+----
+ oc delete namespace <cluster-name>
+----
+This deletes all the namespace-scoped custom resources created for this SNO cluster.
+
+. Remove the cluster's managed cluster resource.
+[source,terminal]
+----
+ oc delete managedcluster <cluster-name>
+----
+
+. Redo the <<custom-resource-creation,Custom Resource Creation>> section for this cluster.
+
+.Additional Resources
+
+Kubernetes Custom Resources explained within the context of Assisted Installer Service.
+
+|===
+| Resource | Explanation | Usage | Scoping | New Resource
+
+| BareMetalHost
+| Contains the connection information for the BMC of the target Bare Metal machine
+| Loads and boots the Discovery ISO on the target machine using the Redfish protocol.
+| Namespace-scoped
+| No*
+
+| InfraEnv
+| Contains information for pulling OCP onto the target Bare Metal machine
+| Used with ClusterDeployment to generate the Discovery ISO for the SNO cluster.
+| Namespace-scoped
+| Yes
+
+| AgentClusterInstall
+| Specifies the SNO cluster's configuration such as networking, number of supervisor (control plane) nodes, etc. Will show the kubeconfig and credentials after completed installation.
+| Used to specify the SNO cluster's configuration information and provide status during the installation of the cluster.
```

Comment:
This is useful information, but I don't think it fits in a procedure. I would be inclined to rewite this content as an "overview of AI" concept module.